app/views/sasaran_program_opds/excel_spip.xlsx.axlsx

```diff
@@ -69,36 +70,35 @@ wb.add_worksheet(name: "SasaranProgramKegiatanOPD") do |sheet|
       
       # sasaran program ( kabid )
       ind_sas_kepala.sasaran_kabid.each do |sas_kabid|
-        sheet.add_row ["#{sas_kabid.sasaran_kinerja}", nil, nil, nil, nil, nil, "#{sas_kabid&.program_kegiatan&.nama_program || 'PROGRAM PENUNJANG URUSAN PEMERINTAHAN DAERAH KABUPATEN/KOTA'}"], offset: 4
+        sheet.add_row ["#{sas_kabid.sasaran_kinerja}", nil, nil, nil, nil, nil, "#{sas_kabid.program_kabid.pop || '-'}"], offset: 4
```

Comment:
**warning!!**
may caused an error 
discuss about the multiple program_kabid behaviour